docs/hub/security-sso.md

```diff
@@ -22,22 +22,38 @@ When SSO is enabled, the following applies:
 - Org members logged in on hf.co must authenticate through SSO even to access public content from your org. Such content will still be accessible to anyone outside the org. SSO only applies to your org, meaning your org members can still belong to other orgs on hf.co.
 - Role mapping is supported and determined by your IdP and your settings. Enabling role mapping means you can't assign roles manually in the org.
 
-## Configuring SSO
+## Supported Identity Providers
 
-To successfully enable SSO on the Hugging Face Hub for your organization, you need to set up an OIDC/SAML application in your IdP. We will provide a callback URL pointing to the `huggingface.co` domain. Once you create the application, you will need to provide us with the following information:
+You can easily integrate Hugging Face Hub with a variety of Identity Providers, such as Okta, OneLogin or Azure Active Directory (Azure AD). Hugging Face Hub can work with any OIDC-compliant or SAML (*SP Initiated*) Identity Provider.
 
-### OIDC
+## How to configure OIDC/SAML provider in the Hub
 
-- Client ID
-- Client Secret
-- Issuer URL
+You can check your different guides to configure your SSO provider, if yours is not available you can consult one of them to give you an idea of how to do it.
```

Comment:
can you reword this @Michellehbn? i don't think the current wording makes sense 😊